packages/backend/serverless-queue/src/utils/queue-handler.ts

```diff
@@ -0,0 +1,65 @@
+import { SQSHandler } from "aws-lambda";
+import { jsonApiError } from "@gravitywelluk/json-api-error";
+import * as R from "ramda";
+
+import { QueueJobStatus, QueueModels } from "../models/queue-job";
+
+import { JobResult, processJob } from "./process-queue-job";
+
+export interface JobData<T extends keyof QueueJobPayloads> extends QueueJobPayloadProcessing {
+  data: QueueJobPayloads[T];
+}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface QueueJobPayloads {}
```

Comment:
it seems we don't need these types anymore since we won't know the the job payload from inside the queue